pkg/server/plugin/nodeattestor/k8s/psat/psat.go

```diff
@@ -15,6 +15,12 @@ import (
 	"github.com/spiffe/spire/pkg/common/plugin/k8s/apiserver"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
+
+	// Add auth providers to authenticate to clusters to verify tokens
+	_ "k8s.io/client-go/plugin/pkg/client/auth/azure"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/openstack"
```

Comment:
Actually, I think all we need is the following, which would be more robust as more providers are added. We'd automatically get new providers when the client-go dependency was updated.

```suggestion
	// Add auth providers to authenticate to clusters to verify tokens
	_ "k8s.io/client-go/plugin/pkg/client/auth"
```

Reply:
you're right - this is better, let me change it to that.